docs/mons.md

```diff
@@ -1,4 +1,6 @@
-# Mons
+# Mon Commands
```

Comment:
this change requires to update in the Readme doc where we are calling mon-endpoints